jupyter_events/logger.py

```diff
@@ -10,9 +10,10 @@
 import typing as t
 import warnings
 from datetime import datetime, timezone
+from importlib.metadata import version
 
 from jsonschema import ValidationError
-from pythonjsonlogger import jsonlogger
+from packaging.version import parse
```

Comment:
this is a novel (though common) dependency: we'll probably ship it on [conda-forge](https://github.com/conda-forge/jupyter_events-feedstock/pull/23/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR26) with it added, but will PR up as well.